nservicebus/azure-service-bus/configuration/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Azure Service Bus Configuration
+summary: Configuring Azure ServiceBus as transport
+tags:
+- Azure
+- Cloud
+- Configuration
+---
+
+## Getting Started
+
+In order to setup a basic endpoint using the Azure Service Bus transport the following configuration settings need to be provided.
+
+snippet:AzureServiceBusTransportGettingStarted
+
+ * `UseTransport<AzureServiceBusTransport>()`: Enables the Azure Service Bus transport.
+ * `UseTopology<ForwardingTopology>()`: Defines what the underlying layout of Azure Service Bus messaging entities looks like.
+ * `ConnectionString()`: Specifies the connection string of the Azure Service Bus namespace to be used by the transport.
+ * `UsePersistence<InMemoryPersistence>()`: The Azure Service Bus transport has no dependency on persistence, so the in memory one will be sufficient. If there is a need to use sagas, one of the other persister implementations needs to be provided.
+
+## Full Configuration API
+
+As from version 7, the Azure Service Bus transport has an extensive, code-only, configuration API. This API can be accessed from the `UseTransport<AzureServiceBusTransport>()` extension method. A full listing of the API can be found on [the configuration API page](/nservicebus/azure-service-bus/configuration/configuration.md)
+
+The configuration API provides fine grained access to the behavior of different layers in the transport.
+
+ * `UseTopology<T>()` controls how NServiceBus will lay out entities in the Azure Service Bus namespace.
+
+In order to control the creation of these entities, the API provides direct access to the underlying configuration description objects through the following extension methods.
+
+ * `Queues()`
+ * `Topics()`
+ * `Subscriptions()`
+
```

Comment:
Would it make sense to link each to the corresponding configuration.md sections? If that would be useful I can add links.